kf_lib_data_ingest/etl/ingest_pipeline.py

```diff
@@ -213,11 +214,12 @@ def run(self):
 
     def _log_count_results(self, all_passed, messages):
         if all_passed:
-            summary = '✅ Count Analysis Passed!\n'
+            summary = '✅  Count Analysis Passed!\n'
```

Comment:
This is a problem in your terminal. Unicode since 9.0 specifies that emoji should be double-wide but you've disabled that. You need to activate "Use Unicode version 9 widths" in iterm2.